ibis/backends/sqlite/__init__.py

```diff
@@ -19,7 +19,6 @@
 from ibis.backends import UrlFromPath
 from ibis.backends.sql import SQLBackend
 from ibis.backends.sql.compilers.base import C, F
-from ibis.backends.sqlite.converter import SQLitePandasData
```

Comment:
Are changes like this still needed? IIUC

- Backends will still require pandas/pyarrow/numpy to work properly
- `ibis.to_sql` and expressions should work without these deps, but that only requires importing the compilers, not the backend itself

If possible, I'd like to avoid imports from within methods if they can remain top-level.

Reply:
I think it worked. Let's see what CI says.